### PR TITLE
Add to README.md file a table of content, default image and installation instructions

### PR DESCRIPTION
**Pull request by Emmanuelcazlo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,56 @@
-# Pokedex
-this is a demo of a pokedex proyect
+<div align="center">
+    <h1>POKEDEX</h1>
+        <img src="https://res.cloudinary.com/dvxtbmch0/image/upload/v1644636956/pokebola_s8c3yd.png" alt="Logo" width="150" height="150" />
+</div>
+<details>
+    <summary>Table of Contents</summary>
```
Comment (pablo179) [inline README.md:6]: very good add

File: README.md [hunk 2]
```diff
@@ -1,2 +1,56 @@
...
+    </ul>
+</details>
+<br>
+
+<div id="about" >
+    <h2 align="justify"> Here's a Pokedex powered by PokeAPI build with React.js, that provides information, stats, movements, evolution chain and fuctions like search or suffle to learn about a random Pokémon.</h2>
+</div>
+
+<br>
+<a align="center"> <img src="https://res.cloudinary.com/dvxtbmch0/image/upload/v1644551448/Captura_de_pantalla_de_2022-02-10_21-50-18_va8kpv.png"  width="1000" /> </a>
```
Comment (pablo179) [inline README.md]: let's use the image of the pokedex that we attempt to copy: https://dribbble.com/shots/15128634-Pokemon-Pokedex-Website-Redesign-Concept

File: README.md [hunk 3]
```diff
@@ -1,2 +1,56 @@
...
+
```
Comment (pablo179) [inline README.md:34]: also we need to add a reference like: this design is made by {author} and we get it from {link}

File: README.md [hunk 4]
```diff
@@ -1,2 +1,56 @@
...
+cd pokedex
+```
+3. Install NPM packages
+```sh
+npm install
+```
+4. Run project
+```sh
+npm start
+```
```
Comment (pablo179) [inline README.md]: we have the `yarn` commando too

**Review by pablo179 — CHANGES_REQUESTED** [review]
nice job, just some tlittle comments

File: README.md (hunk 4 above)
Comment (Emmanuelcazlo) [reply README.md]: that commands only replace npm ?

**Review by pablo179 — APPROVED** [review]
Just fix the conflict on the branch to merge

**Pull request merged**